turftopic/models/cluster.py

```diff
@@ -271,12 +265,12 @@ def fit_predict(
                 console.log(
                     f"Topic reduction done from {n_topics} to {self.n_reduce_to}."
                 )
-                status.update("Reestimating parameters.")
-                self._estimate_parameters(
-                    embeddings,
-                    self.doc_term_matrix,
-                )
-                console.log("Reestimation done.")
+            status.update("Estimating parameters.")
```

Comment:
I see why you did this, but as it stands, it is going to break stuff. The topic reduction step is dependent on parameters that have already been estimated. We should really add some integration tests to prevent stuff like this.